### DISCUSSION TITLE
fa/app/HiddifyCli-guide/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4117] [Content Help کمک به اصلاح محتوا]
# fa/app/HiddifyCli-guide/

The proxy manager platform.

https://hiddify.com/fa/app/HiddifyCli-guide/

**Comment by papampi** [comment]
میشه لطفا آموزش دیتور وارپ و نحوه اضافه کردن لایسنس وارپ برای هایدیفای  کامندلاین هم بزراید

**Reply by TomCat-Hub** [reply]
درود بر شما
وقت بخیر

ممنونم که کامنت گذاشتید. حتما به بخش مربوطه ارجاع داده میشه و در صورت امکان مطلبی در این مورد منتشر میشه. با این حال، تا اون موقع شما میتونید هر سوالی داشتید اون رو توی گروه تلگرام ما و تاپیک مربوط به سوالتون بپرسید تا دوستان شما رو راهنمایی کنند.

لینک گروه تلگرام هیدیفای:
https://t.me/hiddify_board


ممنونیم که هیدیفای رو انتخاب کردید.
مانا باشید

**Comment by cruldra** [comment]
How to modify the binding address of webui?it can only be accessed on localhost

**Reply by czd798** [reply]
同样的问题，没地方改bind

**Comment by mehrdad1985** [comment]
امکان اجرای کلاین در vpn mode وجود ندارد . چنانچه این امکان اضافه گردد و در containerهم قابل اجرا باشد کاملا در میکروتیک قابل استفاده خواهد بود .

**Comment by hosseinbaghi** [comment]
سلام وقت بخیر
اگه به hiddifycli لینک سابسکریپشن اضافه کنم چجوری میتونم کانفیگ هارو دستی انتخاب کنم؟
